src/Backups/BackupsWorker.cpp

```diff
@@ -740,6 +731,34 @@ std::vector<BackupsWorker::Info> BackupsWorker::getAllActiveRestoreInfos() const
     return res_infos;
 }
 
+bool BackupsWorker::checkConcurrentBackups(const BackupSettings & backup_settings) const
```

Comment:
Not really important, but the name `hasConcurrentBackups()` seems a bit better. Because this function doesn't throw an exception and only returns true or false. If the name starts with `check` that is not obvious, and if the name starts with `is` or `has` it's more clear.

Reply:
I prefer `checkConcurrentBackups` over `has` , because concurrent backups are allowed if `allow_concurrent_backups` is set to true. I can move the addInfo & throwing exception inside the `checkConcurrentBackups`  function if it helps with the return value confusion. what do you think?

Reply:
For me both ways are ok:

```
void doBackup()
{
...
bool has_concurrent_backups = hasConcurrentBackups()
addInfo(...)
if (has_concurrent_backups)
  throw ...
```

or

```
void doBackup()
{
...
if (!allow_concurrent_backups)
  checkNoConcurrentBackups(...)
```

Reply:
If you prefer the second way please write in the `checkNoConcurrentBackups()` function an explanation why do you call addInfo() there.

Reply:
if we remove allow_concurrent_backups outside we can call is hasConcurrentBackups(). I like that idea, it makes it more generic so the function can be reused later. 
How about

```
void doBackup()
{
...
if (!allow_concurrent_backups && hasConcurrentBackups())
  {
        addInfo
        throw
  }
}
```

Reply:
I will add a comment for addInfo as you suggested

Reply:
addressed it